traffic_ops/app/db/migrations/20190424165437_valid_latitudes_and_longitudes.sql

```diff
@@ -0,0 +1,35 @@
+/*
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+        http://www.apache.org/licenses/LICENSE-2.0
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+
+-- Database constraint for valid lat-long values
+
+-- +goose Up
+CREATE DOMAIN latitude AS numeric CHECK (VALUE >= -90 AND VALUE <= 90);
+CREATE DOMAIN longitude AS numeric CHECK (VALUE >= -180 AND VALUE <= 180);
+
+UPDATE coordinate SET latitude=0 WHERE latitude>90 OR latitude<-90;
```

Comment:
do we have values outside of the acceptable ranges? changing data like this seems problematic although i guess if outside the range not really valid anyhow but now you've changed invalid values (ie 200) to a valid value (0) and the problem has been masked imo.

Reply:
Well, coordinates that don't exist on Earth can't be used correctly in any context. So I don't think it's problematic to change them - but what I could do instead is coerce values above the maximum down to the maximum, and values below the minimum up to the minimum. Would that be better than just setting everything to zero.